### PR TITLE
WIP: Pelumi/update/sdk docs

### PR DESCRIPTION
**Pull request by Oluwadaminiola**
this pr is for the updated design for SDKs and the app portal

### CODE REVIEW

**Comment by horlah** [comment]
Screenshot 2022-03-22 at 6 28 56 PM

The links in this section are leading to the SaaS. @Oluwadaminiola

File: web/ui/website/assets/images/sdk.svg
Comment (horlah) [inline web/ui/website/assets/images/sdk.svg:9]: Please replace this with a png image. Base 64 string in SVG images takes longer to load most times, and considering they'll likely open this on a phone or tab.

**Review by horlah — CHANGES_REQUESTED** [review]

**Comment by horlah** [comment]
Screenshot 2022-03-22 at 8 44 53 PM

Please check the space here against JK's design. @Oluwadaminiola

File: web/ui/website/layouts/docs.vue
Comment (horlah) [inline web/ui/website/layouts/docs.vue:28]: I'm not sure I understand why these sub-links here are hidden. @Oluwadaminiola

File: web/ui/website/layouts/docs.vue
Comment (horlah) [inline web/ui/website/layouts/docs.vue:40]: Hmm, nice hack here. We should check it out.

File: web/ui/website/pages/docs/index.vue
Comment (horlah) [inline web/ui/website/pages/docs/index.vue:29]: This is no longer needed right?

**Review by horlah — COMMENTED** [review]
Cool stuff here @Oluwadaminiola, I think we should review the logic for the SDK page.

File: web/ui/website/assets/images/sdk.svg
Comment (Oluwadaminiola) [reply web/ui/website/assets/images/sdk.svg:9]: okay noted

File: web/ui/website/pages/docs/index.vue
Comment (horlah) [inline web/ui/website/pages/docs/index.vue:85]: Please delete this @Oluwadaminiola

**Review by horlah — APPROVED** [review]
Please rename this PR before merging

**Pull request merged**
